python/src/diffusionkit/mlx/model_io.py

```diff
@@ -710,6 +718,13 @@ def load_flux(
             hidden_size=config.hidden_size,
             mlp_ratio=config.mlp_ratio,
         )
+    elif model_key == "FLUX.1-dev":
```

Comment:
Shall we consolidate this under `if model_key in ["FLUX.1-schnell", "FLUX.1-dev"]` and dedupe code?